crates/mvr-cli/src/lib.rs

```diff
@@ -536,7 +544,50 @@ fn insert_root_dependency(
         move_section.insert(LOCK_PACKAGE_VERSION_KEY, value(3));
     }
 
-    Ok(doc.to_string())
+    Ok(doc)
+}
+
+/// For `local` dependencies of a given package, we want to convert them into `git` dependencies,
+/// where the `git` dependency's `repository_url` is the parent package's `repository_url`,
+/// the `tag` is the parent package's `tag`, and the `path` is the parent package's `path` joined
+/// with the `local` dependency's `path`.
+///
+/// Before:
+///
+/// id = "dep"
+/// source = { local = "../token" }
+/// After:
+///
+/// id = "dep"
+/// source = { git = "https://github.com/mvr-test/parent-package.git", rev = "v1.0.0", subdir = "packages/parent-package-dir/token" }
+///
```

Comment:
````suggestion
/// ```
/// id = "dep"
/// source = { local = "../token" }
/// ```
///
/// After:
/// ```
/// id = "dep"
/// source = { git = "https://github.com/mvr-test/parent-package.git", rev = "v1.0.0", subdir = "packages/parent-package-dir/token" }
///```
````

Reply:
Not sure how the backticks got dropped